common/changes/@microsoft/rush/readable-cache-hash_2021-08-04-16-53.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add project, command name, and file extension to build-cache files",
+      "type": "none"
```

Comment:
Because this change does change the output of build cache files, and also invalidates any existing cache entries for the current repo, I'd suggest this be at least `patch` -- maybe even `minor`.

Reply:
Actually Rush is published using a version policy where the bump is determined at publishing time. (By editing the [nextBump](https://github.com/microsoft/rushstack/blob/5f5ff298f5cf05f2ab74745812bb6c466fd3add7/common/config/rush/version-policies.json#L94) setting.)

(So the name `"none"` is confusing here.  Really it means "not applicable."  I've always wanted to change it, because there is a "none" answer for `rush change` that means NOT to create a change file at all.)

Reply:
Ahhh ok, this wasn't a choice by the author - gotcha.